pkgs/by-name/kr/krop/package.nix

```
@@ -1,6 +1,32 @@
-{ lib, fetchFromGitHub, python3Packages, libsForQt5, ghostscript, qt5}:
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  libsForQt5,
+  ghostscript,
+  qt5,
+  fetchPypi,
+}:
 
-python3Packages.buildPythonApplication rec {
+let
+  localPython = python3.override {
```

```suggestion
  py = python3.override {
```
nit: we normally name this like that